.coderabbit.yaml

```
@@ -0,0 +1,41 @@
+  # Review new commits in existing PRs
+  review_status: true
+  
+  # High-level summary of the PR
+  high_level_summary: true
+  
+  # Poem about the PR (fun feature)
+  poem: false
+  
+  # Review effort estimate
+  review_effort: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)